__tests__/App.test.jsx

```diff
@@ -29,12 +29,26 @@ describe('<App/>', () => {
     const component = shallow(<App name="test"/>)
 
     expect(component.instance().props.name).toBe('test')
-  })
-  describe('<MyComponent />', () => {
-    it('renders three <Foo /> components', () => {
-      const wrapper = mount(<App />);
-      console.log(wrapper.debug())
-      expect(wrapper.find('Form').length).toBe(1);
-    });
-  })
+  });
+
+  it('toggles the state is Clicked', () => {
+    const wrapper = create(<App />);
+    const instance = wrapper.getInstance();
+    console.log(instance.state.isClicked)
+    expect(instance.state.isClicked).toEqual(false);
+    instance.changeToVisible();
+    expect(instance.state.isClicked).toEqual(true);
+    instance.changeToVisible();
+    expect(instance.state.isClicked).toEqual(false);
+  });
+
+  it('toggles the state is Clicked for Guests', () => {
+    const wrapper = create(<App />);
+    const instance = wrapper.getInstance();
+    expect(instance.state.guestsClicked).toEqual(false);
+    instance.changeGuestsVisible();
+    expect(instance.state.guestsClicked).toEqual(true);
+    instance.changeGuestsVisible();
+    expect(instance.state.guestsClicked).toEqual(false);
+  });
```

Comment:
Maybe it's the way github is presenting the changes, but it really looks like there are some broken parenthesis here...  Or do you not need the 'it' to be within the describe for it to run?

Reply:
I think this is how github presents it. You should have 'it' inside describes, otherwise you will get error.